tests_common/copy_files_from_core.py

```diff
@@ -52,8 +52,8 @@ def cleanup_dir(directory: str):
 simple_project_dir = os.path.join(data_dir, "simple_project")
 db_common_py = os.path.join(db_dir, "common.py")
 dest_db_common_py = os.path.join(dest_db_dir, "common.py")
-dump_tool_py = os.path.join(db_dir, "dump_tool.py")
-dest_dump_tool_py = os.path.join(dest_db_dir, "dump_tool.py")
+dump_tool_py = os.path.join(db_dir, "migration_tests", "dump_tool.py")
```

Comment:
Executing this command results in a failure, because the `migration_tests` directory doesn't exist.